insights/parsers/sctp.py

```diff
@@ -254,3 +257,72 @@ def search(self, **args):
                 when args search do not match then it returns `[]`.
         """
         return keyword_search(self.data, **args)
+
+
+@parser(Specs.sctp_snmp)
+class SCTPSnmp(Parser, dict):
+    """
+    This parser parses the content of ``/proc/net/sctp/snmp`` file,
+    which contains statistics related to SCTP states, packets and chunks.
+
+    Sample content::
+
+        SctpCurrEstab                   	5380
+        SctpActiveEstabs                	12749
+        SctpPassiveEstabs               	55
+        SctpAborteds                    	2142
+        SctpShutdowns                   	5295
+        SctpOutOfBlues                  	36786
+        SctpChecksumErrors              	0
+        SctpOutCtrlChunks               	1051492
+
+    Data is stored in a dictionary.
+
+    Examples:
+
+        >>> type(sctp_snmp)
+        <class 'insights.parsers.sctp.SCTPSnmp'>
+        >>> sctp_snmp.get('SctpCurrEstab')
+        5380
+        >>> sctp_snmp.get('SctpChecksumErrors') == 0
+        True
+        >>> sctp_snmp.stats
+        {'SctpCurrEstab': 5380, 'SctpActiveEstabs': 12749, 'SctpPassiveEstabs': 55, 'SctpAborteds': 2142, 'SctpShutdowns': 5295, 'SctpOutOfBlues': 36786, 'SctpChecksumErrors': 0, 'SctpOutCtrlChunks': 1051492}
```

Comment:
Due to the different hash algorithms of python 2 and 3, the order of elements of the `dict`/`set` will be different. And here the test will only compare the converted string which is in a fixed order, if it passes in python 3, it will fail in python 2.

Would you please change it to compatible with all python versions? e.g.
```python
>>> len(sctp_snmp.stats)
8
>>> sctp_snmp.stats.get('SctpCurrEstab')
5380
```
Or `sorted` them and compare with `==`.

Reply:
Added new examples : 
https://github.com/RedHatInsights/insights-core/pull/2298/files#diff-1a1ec796e7a602ccbea2a1fda0999715R289-R294

List in output is already sorted